### ISSUE TITLE
Demo plugin a bit too trivial, add edit text demo?

### ISSUE THREAD

**Issue by BdR76** [kbilsted/NotepadPlusPlusPluginPack.Net#67]
This is an excellent template but the demo plugin is a bit too trivial imho. I mean a plug-in would typically do something with the current opened text file.

So instead of a line-jump demo, maybe also add word find and replace demo. So two textboxes and a button to replace one with the other, for example "Cat" and "Kitten" or something like that.

**Comment by mahee96** [comment]
Yeah that sounds nice, but if you want something for a quickly, you could look upto scintilla and notepad ++ APIs from the notepad++ repository coz this project should act just as a bridge so all APIs supported by notepad++ should be supported by this project too.
